pkg/controller/master/upgradelog/controller.go

```
@@ -750,19 +751,20 @@ func (h *handler) stopCollect(upgradeLog *harvesterv1.UpgradeLog) error {
 	var err error
 	err = h.clusterFlowClient.Delete(util.HarvesterSystemNamespaceName, name.SafeConcatName(upgradeLog.Name, util.UpgradeLogFlowComponent), &metav1.DeleteOptions{})
 	if err != nil && !apierrors.IsNotFound(err) {
-		return err
+		return fmt.Errorf("failed to delete clusterflow %s/%s", util.HarvesterSystemNamespaceName, name.SafeConcatName(upgradeLog.Name, util.UpgradeLogFlowComponent))
```

Consider propagating the actual error message up instead of swallowing it. In case there is an error having the root error in the log may help diagnosis.

---

o, the error is missing, needs to be added back